### DISCUSSION TITLE
Ideas for new board designs

### DISCUSSION THREAD

**Discussion by hape42** [hape42/dailygammon#90] [Ideas]
From version 1.4 it has become a bit easier to create new designs for boards.

There are now virtually no limits to creativity.

How this works with a new design Karlheinz has described [here](https://github.com/hape42/dailygammon/wiki/New-Boards) very well.

It is not very likely but it did happen. Two had exactly the same basic idea. Of course, that's a pity for the time invested. Therefore, we describe here briefly an idea for a new board.

You can also have an idea and look for someone who has time to implement this idea.
This should then be clearly expressed in the idea.

For each new board, a directory with a number must be created in the asset catalog.

In this thread you can reserve a number.

Until now the following numbers are assigned:

<table align="center">
  <tr>
    <th>#</th>
    <th>name</th>
     <th>Designer</th>
    <th>Status</th>
 </tr>
  <tr>
    <td align="center">1</td>
    <td align="center">Classic Original</td>
    <td align="center">DailyGammon</td>
    <td align="center">published</td>
 </tr>
  <tr>
    <td align="center">2</td>
    <td align="center">Classic HD</td>
    <td align="center">DailyGammon</td>
    <td align="center">published</td>
 </tr>
  <tr>
    <td align="center">3</td>
    <td align="center">Blue / White Original</td>
    <td align="center">DailyGammon</td>
    <td align="center">published</td>
 </tr>
  <tr>
    <td align="center">4</td>
    <td align="center">Red / Grey HD</td>
    <td align="center">hape42</td>
    <td align="center">published</td>
 </tr>
  <tr>
    <td align="center">5</td>
    <td align="center">Wood HD</td>
    <td align="center">darkhelmet</td>
    <td align="center">published</td>
 </tr>
  <tr>
    <td align="center">6</td>
    <td align="center">Metal HD</td>
    <td align="center">darkhelmet</td>
    <td align="center">published</td>
 </tr>
  <tr>
    <td align="center">7</td>
    <td align="center">Mono HD</td>
    <td align="center">darkhelmet</td>
    <td align="center">published</td>
 </tr>
  <tr>
    <td align="center">8</td>
    <td align="center">Unicorn</td>
    <td align="center">Jutta Schneider</td>
    <td align="center">published</td>
 </tr>
  <tr>
    <td align="center">9</td>
    <td align="center">Golf</td>
    <td align="center">Jutta Schneider</td>
    <td align="center">work in progress</td>
 </tr>
  <tr>
    <td align="center">10</td>
    <td align="center">Snooker</td>
    <td align="center">Jutta Schneider</td>
    <td align="center">work in progress</td>
 </tr>
  <tr>
    <td align="center">11</td>
    <td align="center">Billiard</td>
    <td align="center">Jutta Schneider</td>
    <td align="center">work in progress</td>
 </tr>
  <tr>
    <td align="center">12</td>
    <td align="center"> Steampunk </td>
    <td align="center"> darkhelmet </td>
    <td align="center">work in progress</td>
 </tr>
  <tr>
    <td align="center">13</td>
    <td align="center">no name yet, a nice traditional board with good colors (suggestions for a real life board with great colors?)</td>
    <td align="center">darkhelmet</td>
    <td align="center">idea</td>
 </tr>
  <tr>
    <td align="center">14</td>
    <td align="center">Earth and Moon and Space (will need to find a shorter name)</td>
    <td align="center">darkhelmet</td>
    <td align="center">experimenting</td>
 </tr>
</table>

**Comment by kagsteiner** [comment]
I would like to reserve

- board 12 as "Steampunk"
- board 13 with no name yet, a nice traditional board with good colors (suggestions for a real life board with great colors?)

**Comment by hape42** [comment]
Ideas: Themes for soccer, football, basketball, tennis 

Unfortunately, I only have the ideas but no time to implement it.

If someone would like to implement please just briefly report here.

**Comment by kagsteiner** [comment]
I'm not sure if this will work out, but I'd like to do a board with earth and moon as checkers and some space-y images for background and checkers. Sadly, Earth and Moon don't look so great in small checkers.
<img width="529" alt="Bildschirm­foto 2023-02-01 um 19 49 42" src="https://user-images.githubusercontent.com/25194620/216135711-cbaeebea-78dc-471d-ab62-0a8295f6df5e.png">

This is too much of everything, but overall this is what I would like to do. What do you think? Worthwhile?

**Reply by hape42** [reply]
I like it
